packages/starknet-snap/src/index.ts

```diff
@@ -28,41 +56,37 @@ import {
   STARKNET_SEPOLIA_TESTNET_NETWORK,
   STARKNET_TESTNET_NETWORK,
 } from './utils/constants';
-import { dappUrl, upsertErc20Token, upsertNetwork, removeNetwork } from './utils/snapUtils';
-import { getStoredNetworks } from './getStoredNetworks';
-import { getStoredTransactions } from './getStoredTransactions';
-import { getTransactions } from './getTransactions';
-import { recoverAccounts } from './recoverAccounts';
-import { Mutex } from 'async-mutex';
-import { ApiParams, ApiRequestParams } from './types/snapApi';
-import { estimateAccDeployFee } from './estimateAccountDeployFee';
-import { executeTxn } from './executeTxn';
-import { estimateFees } from './estimateFees';
-import { declareContract } from './declareContract';
-import { signDeclareTransaction } from './signDeclareTransaction';
-import { signDeployAccountTransaction } from './signDeployAccountTransaction';
-import { upgradeAccContract } from './upgradeAccContract';
+import { getAddressKeyDeriver } from './utils/keyPair';
 import { logger } from './utils/logger';
-import { getStarkName } from './getStarkName';
-
-import type { OnRpcRequestHandler, OnHomePageHandler, OnInstallHandler, OnUpdateHandler } from '@metamask/snaps-sdk';
-import { InternalError, panel, row, divider, text, copyable } from '@metamask/snaps-sdk';
-import { ethers } from 'ethers';
-import { getBalance, getCorrectContractAddress, getKeysFromAddressIndex } from './utils/starknetUtils';
+import { toJson } from './utils/serializer';
+import {
+  dappUrl,
+  upsertErc20Token,
+  upsertNetwork,
+  removeNetwork,
+} from './utils/snapUtils';
+import {
+  getBalance,
+  getCorrectContractAddress,
+  getKeysFromAddressIndex,
+} from './utils/starknetUtils';
+import { verifySignedMessage } from './verifySignedMessage';
 
 declare const snap;
 const saveMutex = new Mutex();
 
-export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) => {
+export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
```

Comment:
Why do we remove `origin` in here ? Is this part of the eslint style ?

Reply:
yes, unused variable